docker-compose.yml

```diff
@@ -21,10 +21,10 @@ services:
     build: 
       context: .
       args:
-        SHIELDHIT_PATH: ${SHIELDHIT_PATH}
+        SHIELDHIT_PATH: ${SHIELDHIT_PATH-/shieldhit}
```

Comment:
What does it mean ? Is it path concatenation ? Or something else ?

Reply:
@retkiewi